iocore/net/OCSPStapling.cc

```diff
@@ -1097,7 +1204,21 @@ stapling_refresh_response(certinfo *cinf, TS_OCSP_RESPONSE **prsp)
     goto err;
   }
 
-  *prsp = query_responder(cinf->uri, cinf->user_agent, req, SSLConfigParams::ssl_ocsp_request_timeout);
+  if (SSLConfigParams::ssl_ocsp_request_mode) { // True: GET, False: POST
```

Comment:
This also would be clearer as `ssl_ocsp_prefer_get` or some such. Otherwise maybe create an enum and switch on that value?

```cpp
  bool use_get = false;
  switch(SSLConfigParams::ssl_ocsp_request_mode) {
    case USE_GET:
      use_get = true;
  }

Reply:
We can stick with things as is for now and tweak this if we add new modes.